recipes/trinity/meta.yaml

```diff
@@ -1,44 +1,52 @@
 about:
-    home: 'https://github.com/trinityrnaseq/trinityrnaseq'
-    license: "https://raw.githubusercontent.com/trinityrnaseq/trinityrnaseq/master/LICENSE.txt"
-    summary: "Trinity assembles transcript sequences from Illumina RNA-Seq data"
+  home: https://github.com/trinityrnaseq/trinityrnaseq/
+  license: ''
+  summary: 'Trinity RNA-Seq de novo transcriptome assembly'
+
 build:
-  number: 6
+  number: 1
   skip: True # [osx]
+
 package:
   name: trinity
-  version: '2.1.1'
-source:
-  fn: v2.1.1.tar.gz
-  md5: 45c1ad13c331d6a1976f6a9b5b015daa
-  url: https://github.com/trinityrnaseq/trinityrnaseq/archive/v2.1.1.tar.gz
+  version: '2.2.0'
+
 requirements:
   build:
-    - perl-threaded
+    - libgcc
```

Comment:
Also, what do you want to do with the 2.1.1 build of trinity? It doesn't build on the bioconda docker image, or at least I couldn't get it to build. How is it in there? 

I could update that recipe. Its probably fine for people who have zlib installed as system dependency, so using zlib as an anaconda dependency wouldn't be too bad.